<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IngredientWindow</class>
 <widget class="QDialog" name="IngredientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBarSugar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelSugar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sugar Tank:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMilk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk Tank:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarMilk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoffee">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee Tank:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarCoffee">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarCacao">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelCacao">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cacao Tank:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSugarAmount">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sugar Tank:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoffeeAmount">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee Tank:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMilkAmount">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk Tank:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCacaoAmount">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cacao Tank:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
